kettle/stream.py

```diff
@@ -0,0 +1,271 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Receive push events for new builds and upload rows to BigQuery.
+"""
+
+from __future__ import print_function
+
+import argparse
+import json
+import os
+import pprint
+import sys
+import traceback
+import time
+
+try:
+    from google.cloud import bigquery
+    from google.cloud import pubsub
+    import google.cloud.exceptions
+except ImportError, e:
```

Comment:
Use `as` not `,`

https://google.github.io/styleguide/pyguide.html?showone=Exceptions#Exceptions